src/main/scala/loamstream/loam/LoamScript.scala

```diff
@@ -2,44 +2,141 @@ package loamstream.loam
 
 import java.nio.file.Path
 
+import scala.util.Success
+import scala.util.Try
+
+import loamstream.compiler.LoamFile
 import loamstream.loam.LoamScript.LoamScriptBox
 import loamstream.loam.LoamScript.scriptsPackage
-import loamstream.util.DepositBox
+import loamstream.util.Files
 import loamstream.util.Sequence
 import loamstream.util.StringUtils
-import loamstream.util.ValueBox
+import loamstream.util.Tries
 import loamstream.util.code.ObjectId
 import loamstream.util.code.PackageId
+import loamstream.util.code.RootPackageId
 import loamstream.util.code.ScalaId
 import loamstream.util.code.SourceUtils
-import scala.util.Try
-import loamstream.util.Tries
-import scala.util.Success
 
 /** A named Loam script */
+sealed trait LoamScript {
```

Comment:
A comment or two detailing the relationship between `LoamFile`, `LoamScript`, etc. would be helpful. Should I use one over another? `LoamFile` doesn't (obviously) inherit from `LoamScript`, but they seem close (naming-wise) in what I would expect them to do?

Reply:
I'll write something up here.  They're related, but also not.